src/vs/workbench/browser/parts/titlebar/menubarControl.ts

```diff
@@ -436,6 +443,43 @@ export class MenubarControl extends Disposable {
 		});
 	}
 
+	private detectAndRecommendCustomTitlebar(): void {
+		if (!isLinux) {
+			return;
+		}
+
+		if (!this.storageService.getBoolean('menubar/electronFixRecommended', StorageScope.GLOBAL, false)) {
+			if (this.currentMenubarVisibility === 'hidden' || this.currentTitlebarStyleSetting === 'custom') {
+				// Issue will not arise for user, abort notification
+				return;
+			}
+
+			const message = nls.localize('menubar.electronFixRecommendation', 'If you experience a low-contrast issue with the menu bar, we recommend trying out the custom title bar.');
+			this.notificationService.prompt(Severity.Info, message, [
+				{
+					label: nls.localize('tryIt', 'Try it'),
```

Comment:
How about just opening the settings editor? "Try it" implies that I can try this out with an easy way to get back but here the user won't know how to undo this.

Reply:
Or change the string to something that sounds more permanent